### ISSUE TITLE
Crashing many times: Exception Error

### ISSUE THREAD

**Issue by molitar** [fmd-project-team/FMD#555]
I been having a lot of crashing lately so I have copied the exception error below.

> Program exception!
Application       : Free Manga Downloader
Version           : 0.9.167.1
Product Version   : Win64
Host Machine      : Windows 10 64-bit
FPC Version       : 3.0.4
LCL Version       : 1.8.4.0
WidgetSet         : Win32/Win64
Target CPU-OS     : x86_64-Win64
Build Time        : 2019/09/21 15:54:42
Path              : C:\DOS\fmd\fmd.exe
Process ID        : 15028
MainThread ID     : 103160
Thread ID         : 33480
Time              : 11/10/2019 13:10:17.899
Sender Class      : TFavoriteThread
Exception Class   : EAccessViolation
Message           : Access violation
  $000000010000AA5A
  $000000010000AAE2
  $00000001004630FF
  $00000001004658FB
  $0000000100465C79
  $000000010027E98B
  $00000001000542ED
  $0000000100003B96
  $0000000100015067
  $00007FFF50657BD4
  $00007FFF51BCCED1

>

**Comment by Slasar41** [comment]
Is cf_bypass.exe still in there?

**Comment by molitar** [comment]
Yes and works fine as I am downloading daily on Kissmana.. Have to manually view manga once in a while so it opens in Kissmanga to refresh Cloudflare so it's there and working.

**Comment by Tmp341** [comment]
Have you tried running fmd with admin rights?

**Comment by molitar** [comment]
Well I have tried running with admin rights. My C drive is my best drive as it's a 512GB SSD drive.  My other 2 drives are large WD Green storage drives at 4TB and 6TB in size. These are slow Green drives for storage.

**Comment by molitar** [comment]
Ok HeavenManga seems to have a severe problem. I have to keep doing manga info several times to get it to refresh chapters.  I have determined that when this happens if the favorites check runs then it crashes with exception.  It is when more then one mangasite has to be updated.

Heaven Manga has to be manually refreshed a good 3 to 4 times a day. Going to try to find another site that carries the two manhwa and not use this site if possible. Too many crashes.
